src/cli.cr

```diff
@@ -40,6 +167,179 @@ CLI.run(
           },
         },
       },
+      user: {
+        commands: {
+          add: {
+            info:      "add a user",
+            action:    "DppmRestApi::CLI.add_user",
+            variables: {
+              name: {
+                info: "human-readable name of the user. Use quotes if you want" +
+                      %<to have spaces in the name, like name="Scott Boggs">,
+              },
+              groups: {
+                info: "comma-separated list of group IDs for which the user \
+                       should have access.",
+              },
+            },
+          },
+          edit: {
+            info: <<-HERE,
+            edit one or more user's groups
+
+            The users may be selected either by sepecifying a user's API key,
+            if it is known, or by matching the name and groups the user is a
+            member of. It is not recommended to match only by name or by groups,
+            as your command may unintentionally affect similar users. For
+            example, two users may choose the same name, but be members of
+            different groups. This doesn't matter as they're only identified
+            internally by their API keys, but you may unintentionally add/remove
+            groups from those users when using this command in that situation.
+
+            Ideally, you won't have multiple users with the same name, or you
+            can use the web interface to edit these values using API keys for
+            exact identification.
```

Comment:
Absolutely too long, users won't want to have their terminals taken with textual information.
This documentation has its place in a `man` or a proper documentation place (web, markdown...)

Reply:
Updated as requested